### DISCUSSION TITLE
Getting a weird character

### DISCUSSION THREAD

**Discussion by esmaeilmirzaee** [starship/starship#3226] [Q&A] [answered]
Hi

I just wondered what are those unwanted character behind the caret `->`. Could somebody help me?

![image](https://user-images.githubusercontent.com/8766916/141272719-179869ba-11b9-43d5-a45d-1cb7218cdb56.png)

**Comment by andytom** [comment, marked as answer]
[From the FAQ](https://starship.rs/faq/#i-see-symbols-i-don-t-understand-or-expect-what-do-they-mean)

> If you see symbols that you don't recognise you can use `starship explain` to explain the currently showing modules.

That should help you find what module these symbols these symbols are coming from so you can either disable the module or change the symbols to something more meaningful to you.
